core-web/libs/block-editor/src/lib/components/dot-block-editor/dot-block-editor.component.stories.ts

```diff
@@ -20,9 +22,7 @@ import {
 } from '@dotcms/data-access';
 import { DotSpinnerModule } from '@dotcms/ui';
 
-import { DotBlockEditorComponent } from './dot-block-editor.component';
-
-import { BlockEditorModule } from '../../block-editor.module';
+import { DotAiServiceMock } from '../../../../../data-access/src/lib/dot-ai/dot-ai-service.mock';
```

Comment:
should we import this as `@dotcms/data-access`

Reply:
Fixed!